catalog/rest/catalog-rest-endpoint/pom.xml

```diff
@@ -193,6 +193,12 @@
             <artifactId>catalog-rest-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.sun.activation</groupId>
+            <artifactId>javax.activation</artifactId>
+            <version>1.2.0</version>
```

Comment:
any reason not to put this in the root pom?

Reply:
`${javax.activation.version}`

Reply:
Just did this, and I missed one. On it now. Thanks!